src/lib/config/providers/ConfigProvider.tsx

```
@@ -0,0 +1,123 @@
+  schemas: T;
+  children: ReactNode;
+}
+
+interface ConfigContextValue<T extends ConfigSchemas> {
+  common: ValidatedConfigs<T>['common'];
+  client: ValidatedConfigs<T>['client'];
+  isLoading: boolean;
+  error: string | null;
+}
+
+const ConfigContext = React.createContext<ConfigContextValue<any> | null>(null);
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)